src/jdk.jfr/share/classes/jdk/jfr/events/ContainerMemoryUsageEvent.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2021, DataDog. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+package jdk.jfr.events;
+
+import jdk.jfr.Category;
+import jdk.jfr.DataAmount;
+import jdk.jfr.Description;
+import jdk.jfr.Enabled;
+import jdk.jfr.Label;
+import jdk.jfr.Name;
+import jdk.jfr.Period;
+import jdk.jfr.StackTrace;
+import jdk.jfr.Threshold;
+import jdk.jfr.Timespan;
+import jdk.jfr.internal.Type;
+
+@Name(Type.EVENT_NAME_PREFIX + "ContainerMemoryUsage")
+@Label("Container Memory Usage")
+@Category({"Operating System", "Container", "Memory"})
+@Description("Container memory usage related information.")
+public final class ContainerMemoryUsageEvent extends AbstractJDKEvent {
+    @Label("Memory Pressure")
+    @Description("(attempts per second * 1000), if enabled, that the operating system tries to satisfy a memory request for any " +
```

Comment:
This unit seems a bit strange. Do we really need to multiply by 1000?

Reply:
This is taken as reported by cgroups - I didn't want to change the semantics so it does not confuse people familiar with cgroups.

Reply:
I don't see this event field being set anywhere? Which Metrics API call is this using?

Reply:
How does it look in proc?

Reply:
This was based on https://github.com/openjdk/jdk/blob/da860290c2657c0fb1de8c77c8dffdb35f1cf938/src/java.base/linux/classes/jdk/internal/platform/CgroupV1Metrics.java#L149

However, that metric is  'internal'  only so it really does not make sense to keep it here and is there only by mistake.

Reply:
Removed